cli/flox/doc/flox-config.md

```diff
@@ -116,11 +116,26 @@ flox config --set 'trusted_environments."owner/name"' trust
     * "hide-all": disables the modification of the shell prompt
     * "hide-default": filters out environments named 'default' from the shell prompt
 
+`state_dir`
+:   Directory where flox should store data that's not critical but also
+    shouldn't be able to be freely deleted like data in the cache directory.
+    (default: `$XDG_STATE_HOME/flox` e.g. `~/.local/state/flox`)
+
 `trusted_environments`
 :   Remote environments that are trusted for activation.
     Contains keys of the form `"<owner>/<name>"` that map to either `"trust"` or
     `"deny"`.
 
+`upgrade_notifications`
+:   Print notification if upgrades are available on `flox activate`.
+    The notification message is:
+
+    Upgrades are available for packages in 'environment-name'.
+
+    Use 'flox upgrade' to get the latest.
```

Comment:
@ysndr @jennymahmoudi  is this what was meant by `Users will be able to search the exact upgrade notification message to find this config setting in docs.` from https://github.com/flox/flox/issues/2534? I think it's a little weird to put it in the man page but it makes sense that we want it show up in search

Reply:
yes! sorry for missing this last week!